### DISCUSSION TITLE
Personal portfolio

### DISCUSSION THREAD

**Discussion by tsheringphuntsho18** [kevinlin1/just-the-class#41] [Q&A]
Please provide feedback.

https://www.figma.com/proto/VUHliEGQn8Q1xV5D0qhmuK?node-id=0-1&mode=design&t=iFhdqGf54w7Gssj1-6

**Comment by kevinlin1** [comment]
Hi, I think you meant to post this in the [discussion for your course](https://github.com/rubcstswe/sda101/discussions). This discussion is for the website template, not any particular course.
